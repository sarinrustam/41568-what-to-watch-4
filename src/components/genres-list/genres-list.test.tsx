import * as React from "react";
import * as renderer from "react-test-renderer";
import GenresList from "./genres-list";

const mockGenres: [string] = [`Comedy`];

describe(`Render correctly`, () => {
  it(`should GenreList render correctly`, () => {
    const handlerSetCurrentGenre = jest.fn();
    const tree = renderer.create(
        <GenresList
          genresList={mockGenres}
          currentGenre={mockGenres[0]}
          onSetCurrentGenre={handlerSetCurrentGenre}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
