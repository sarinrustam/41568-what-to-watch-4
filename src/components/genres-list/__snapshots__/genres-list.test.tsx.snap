// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly should GenreList render correctly 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item catalog__genres-item--active"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Comedy
    </a>
  </li>
</ul>
`;
